client/common/src/types.rs

```diff
@@ -93,8 +94,10 @@ impl<K, D: Debug> Proven<K, D> {
 }
 
 /// Proof of file key(s) in the forest trie.
+#[derive(Clone, Encode, Decode, Debug)]
 pub struct ForestProof<T: TrieLayout> {
     /// The file key that was proven.
+    #[codec(skip)]
```

Comment:
Curious why this is added?

Reply:
I believe it is because he encodes/decodes this struct only for the RPC method, in which case it doesn't care about the `proven` field. Is that correct?

Reply:
Yep! That's exactly right.